### DISCUSSION TITLE
Charset?

### DISCUSSION THREAD

**Discussion by rybarova** [stevenvachon/broken-link-checker#214] [General]
Hello,
I want to filter only broken links so I use filter on my powershell "Select-String Broken". But charset of filtered content is not correct because I have there "ÔöťÔöÇBROKENÔöÇ" instead of "├─BROKEN─".  The same problem is if i want to save this output into file. I tried that via UTF8, ASCII, windows 1250 etc. , but problem is +- same.
Can u help me how to fix it on windows, or what charset to use?

**Environment:**
 - OS and version: Windows 10
 - Node.js version: v10.16.3
 - broken-link-checker version: 0.7.8

**Comment by bnosach** [comment]
I'm having the same issue. Output is garbled up:

**Γö£ΓöÇBROKENΓöÇ** http://www.deadlinkcity.com/error-page.asp?e=400 (HTTP_400)
